server.js

```diff
@@ -1,30 +1,131 @@
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+    const welcomePage = {
+      Hello:
+      "Hello there! Here you'll find an API with information about 1 375 Netflix movies and TV-shows!",
+      Routes: [{
+        "/content": "GET the whole array of movies and tv-shows",
+        "/content/id/'id-number": "GET a specific movie/tv-show by it's unique id-number",
+        "/content/title/'title of movie/show": "GET movies/tv-shows by title",
+        "/content/director/'director of movie/show": "GET movies/tv-shows by director",
+        "/content/type/movie": "GET whole array of movies",
+        "/content/type/show": "GET whole array of tv-shows",
+      }]
+    }
+  res.send(welcomePage)
+  });
+
+
+//Endpoint with all available content from API
+
+app.get('/content', (req, res) => {
+ 
+  res.status(200).json({
+    data: netflixData,
+    success: true,
+  });
+  
+});
+
+
+//Endpoint with movie/tv-show id-number
+
+app.get("/content/id/:id", (req, res) => {
+  const { id } = req.params;
+
+  const contentById = netflixData.find(
+    (content) => content.show_id === +id
+  );
+
+  if (contentById) {
+    res.status(200).json({
+      data: contentById,
+      success: true,
+    });
+  } else {
+    res.status(404).json({
+      data: "The movie/tv-show you're looking for is not here.. Are you sure you've entered the correct id-number?",
+      success: false,
+    })
+  }
+        
+});
+
+
+//Endpoint with title of movie/tv-show
+    
+app.get("/content/:title", (req, res) => {
+  const { title } = req.params;
+
+  const contentByTitle = netflixData.filter(
+    (content) => content.title.toLowerCase() === title.toLowerCase()
+  );
+      
+  if (contentByTitle) {
+    res.status(200).json({
+      data: contentByTitle,
+      success: true,
+    });
+  } else {
+      res.status(404).json({
+        data: "The movie/tv-show you're looking for is not here.. Are you sure you've entered the correct title?",
+        success: false,
+      })
+  };
+
+});
+
+
+//Endpoint with director name
+
+app.get("/content/director/:director", (req, res) => {
+  const { director } = req.params;
+
+  const contentByDirector = netflixData.filter(
+    (content) => content.director.toLowerCase().includes(director.toLowerCase())
+  );
+
+  if (contentByDirector) {
+    res.status(200).json({
+      data: contentByDirector,
+      success: true,
+    });
+  } else {
+      res.status(404).json({
+        data: "This is not the director you're looking for.. Try again you shall.",
+        success: false,
+      })
+  };
+        
 });
 
-// Start the server
+
+//Endpoint with type (movies or tv-shows)
+
+app.get("/content/type/:type", (req, res) => {
+  const { type } = req.params;
+
+  const contentByType = netflixData.filter(
+    (content) => content.type.toLowerCase().includes(type.toLowerCase())
+  );
```

Comment:
the /content/type/movie or show works super well in the browswer and it gives me all the movies or all the tv shows. But since the comparison is .includes() it is also possible to enter 'content/type/o' and it will give me all the movies and tv shows since show and movie both contain an o. I would maybe suggest using '===' instead of .includes() for this particular pathparam since it will give you either movies or tv shows, and then maybe handle if someone enters anything else but movie or show :)